guides/common/modules/ref_webhooks-available-events.adoc

```diff
@@ -0,0 +1,45 @@
+[id="webhooks-available-events_{context}"]
+= Available Events
+
+The following are the webhook events available from the {ProjectWebUI}. `Action` events will trigger webhooks only on `success`, so if an action failed, a webhook is not triggered.
```

Comment:
```suggestion
The following table contains a list of webhook events that are available from the {ProjectWebUI}. `Action` events trigger webhooks only on `success`, so if an action failed, a webhook is not triggered.
```

IBM style says to avoid the future tense.